modules/serverless-installing-cli-linux-ibm-z_tarball.adoc

```diff
@@ -0,0 +1,45 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_z_tarball_{context}"]
+
+== Tarball Installation
+
+For Linux distributions, you can download the CLI directly as a `tar.gz` archive.
+
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[CLI].
```

Comment:
. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[kn] CLI.